linkerd/admin/src/main/scala/io/buoyant/linkerd/admin/names/WebDelegator.scala

```diff
@@ -168,7 +168,7 @@ private[admin] class WebDelegator(
   def apply(req: Request): Future[Response] = req.method match {
     case Method.Get => (req.params.get("d"), req.params.get("n")) match {
       case (Some(DtabStr(dtab)), Some(PathStr(path))) =>
-        val DstBindingFactory.Namer(namer) = linker.params[DstBindingFactory.Namer]
+        val DstBindingFactory.Namer(namer) = linker.routers.head.params[DstBindingFactory.Namer]
```

Comment:
I also agree that this is confusing. Is there a reason we can't leave a way to access the namer via the linker? (we could even pass the linker to the router rather than the namer itself)